wallet-webapps-common/src/main/webapp/vue-app/js/WalletUtils.js

```diff
@@ -440,6 +440,10 @@ export function getNetworkLink() {
   }
 }
 
+export function getUrlHostName(url){
```

Comment:
what is this method about ? I think we need to use the etherscan/polygonscan link as it is

Reply:
deleted and used the etherscan/polygonscan link as it is